packages/server-wallet/src/handlers/close-channel.ts

```diff
@@ -1,17 +1,7 @@
-import {Either, left, right, chain, map} from 'fp-ts/lib/Either';
-import {StateVariables} from '@statechannels/wallet-core';
-import {pipe} from 'fp-ts/lib/function';
 import {ChannelId} from '@statechannels/client-api-schema';
 
-import {SignState, signState} from '../protocols/actions';
-import {ChannelState, ChannelStateWithSupported} from '../protocols/state';
 import {WalletError, Values} from '../errors/wallet-error';
 
-import {hasSupportedState, isMyTurn, supported} from './helpers';
-
-type HandlerResult = Either<CloseChannelError, SignState>;
-type StepResult = Either<CloseChannelError, ChannelStateWithSupported>;
-
 export interface CloseChannelHandlerParams {
```

Comment:
New issue: This file ought to be deleted.

- [ ] If `UpdateChannelHandlerParams` is used, it should be re-named and re-exported elsewhere.
- [ ] `UpdateChannelError` should be re-exported elsewhere